docs/Node/ReadLine.md

```diff
@@ -50,33 +98,33 @@ Prompt the user for input on the specified `Interface`.
 #### `setPrompt`
 
 ``` purescript
-setPrompt :: forall eff. String -> Int -> Interface -> Eff (console :: CONSOLE | eff) Interface
+setPrompt :: forall eff. String -> Int -> Interface -> Eff (readline :: READLINE | eff) Interface
 ```
 
 Set the prompt.
 
-#### `createInterface`
+#### `close`
 
 ``` purescript
-createInterface :: forall eff. Completer eff -> Eff (console :: CONSOLE | eff) Interface
+close :: forall eff. Interface -> Eff (readline :: READLINE | eff) Interface
```

Comment:
I realise the old API was like this, but is there any reason to return the `Interface`? Why not just `Unit`?